### PR TITLE
LG-1719 Add description to radio

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [LG-1719 ](https://jira.mongodb.org/browse/LG-1719 )

### For bug fixes, new features & breaking changes

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/radio-group/src/Radio/Radio.styles.ts [hunk 1]
```diff
@@ -14,13 +14,26 @@ export const inputDisplayWrapperClassName =
...
+  grid-template-areas: 'label label' '. description';
+
   & + & {
     margin-top: 8px;
   }
 `;
 
+export const containerSizeStyle: Omit<Record<Size, string>, 'xsmall'> = {
+  [Size.Small]: css`
+    grid-template-columns: ${14 + 8}px auto;
```
Comment (shaneeza) [inline packages/radio-group/src/Radio/Radio.styles.ts:28]: Where are these numbers coming from?

File: packages/radio-group/src/Radio/Radio.tsx [hunk 2]
```diff
@@ -115,8 +117,18 @@ function Radio({
           />
         </div>
 
-        <div>{children}</div>
+        <>{children}</>
       </Label>
+
+      {description && (
+        <Description
+          className={css`
```
Comment (shaneeza) [inline packages/radio-group/src/Radio/Radio.tsx:125]: Can we move these styles to .styles?

File: packages/radio-group/src/Radio/Radio.types.ts [hunk 3]
```diff
@@ -22,4 +22,9 @@ export interface RadioProps
    * Content that will appear inside of Radio.
    */
   children?: React.ReactNode;
+
+  /**
+   * Description text rendered under the label.
+   */
+  description?: string;
```
Comment (shaneeza) [inline packages/radio-group/src/Radio/Radio.types.ts:29]: We can also add this to PropTypes.

File: packages/radio-group/src/Radio/Radio.styles.ts [hunk 4]
```diff
@@ -14,13 +14,26 @@ export const inputDisplayWrapperClassName =
   createUniqueClassName('radio-group');
 export const inputClassName = createUniqueClassName('radio-group');
 
-export const containerMargin = css`
+export const containerStyle = css`
+  display: grid;
+  grid-template-areas: 'label label' '. description';
+
   & + & {
```
Comment (shaneeza) [inline packages/radio-group/src/Radio/Radio.styles.ts:21]: You could also remove this bit of code and replace with flex and gap. This way, teams can use flex-direction.

**Review by shaneeza — COMMENTED** [review]
✨ looking good - I left a few minor comments.

File: packages/radio-group/src/Radio/Radio.tsx [hunk 5]
```diff
@@ -115,8 +117,18 @@ function Radio({
           />
         </div>
 
-        <div>{children}</div>
+        <>{children}</>
       </Label>
+
+      {description && (
+        <Description
```
[the remark this answers is not in the document]
Comment (TheSonOfThomp) [reply packages/radio-group/src/Radio/Radio.tsx:124]: Yep 👍

File: packages/radio-group/src/Radio/Radio.styles.ts (hunk 4 above)
Comment (TheSonOfThomp) [reply packages/radio-group/src/Radio/Radio.styles.ts:21]: Ah good call!!

**Review by shaneeza — APPROVED** [review]

**Pull request merged**